apps/zotonic_mod_backup/src/models/m_backup.erl

```diff
@@ -30,6 +30,8 @@ m_get([ <<"admin_panel">> | Rest ], _Msg, Context) ->
     {ok, {m_config:get_value(mod_backup, admin_panel, Context), Rest}};
 m_get([ <<"daily_dump">> | Rest ], _Msg, Context) ->
     {ok, {m_config:get_value(mod_backup, daily_dump, Context), Rest}};
+m_get([ <<"encrypt_backups">> | Rest ], _Msg, Context) ->
+    {ok, {m_config:get_value(mod_backup, encrypt_backups, Context), Rest}};
```

Comment:
Isn't this a boolean?  Like the ones above also should be?

Reply:
Yes